src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Table.txt

```diff
@@ -0,0 +1,112 @@
+
+>> Table([{a:0, b:false, c:"Hello"}], [{a:1, b:true, c:"World"}])
```

Comment:
Do you need to specify `#SETUP: TableSyntaxDoesntWrapRecords` in this test, since you are relying on this behavior?

Reply:
sounds good better to be explicit